src/user/serializers.py

```diff
@@ -61,6 +61,7 @@ class AuthorSerializer(ModelSerializer):
     total_score = SerializerMethodField()
     university = UniversitySerializer(required=False)
     wallet = SerializerMethodField()
+    status = SerializerMethodField()
```

Comment:
just curious, why do we put this `status` on the `Author` and not the `User`?

Reply:
I don't think we're returning the User object in the Author field